akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1245,7 +1294,26 @@ trait FlowOps[+Out, +Mat] {
    *
    * This is a shorthand for [[concat]]
    */
-  def ++[U >: Out, M](that: Graph[SourceShape[U], M]): Repr[U, Mat] = concat(that)
+  def ++[U >: Out, M](that: Graph[SourceShape[U], M]): Repr[U] = concat(that)
+
+  /**
+   * Connect this [[Flow]] to a [[Sink]], concatenating the processing steps of both.
+   * {{{
+   *     +----------------------------+
+   *     | Resulting Sink             |
+   *     |                            |
+   *     |  +------+        +------+  |
+   *     |  |      |        |      |  |
+   * In ~~> | flow | ~Out~> | sink |  |
+   *     |  |      |        |      |  |
+   *     |  +------+        +------+  |
+   *     +----------------------------+
+   * }}}
+   * The materialized value of the combined [[Sink]] will be the materialized
+   * value of the current flow (ignoring the given Sink’s value), use
+   * [[Flow#toMat[Mat2* toMat]] if a different strategy is needed.
```

Comment:
which is totally okay since they are separate, but thanks for the poking: the Java version had not yet been outfitted with the drawing!